pennylane/_qubit_device.py

```diff
@@ -613,14 +612,7 @@ def statistics(
 
         for m in measurements:
             # TODO: Remove this when all overriden measurements support the `MeasurementProcess` class
-            if m.obs is not None:
-                obs = m.obs
-                obs.return_type = m.return_type
-            elif m.mv is not None:
-                obs = m.mv
-                obs.return_type = m.return_type
-            else:
-                obs = m
+            obs = m.obs or m.mv or m
```

Comment:
What implications, if any, will this have on device developers?

Reply:
I just used a shorthand to do the same as before, but without changing `obs.return_type` because we don't depend on it later anymore. I can be more verbose to check if it's `None`, but like... is `m.obs` ever `0`, `False`, or something else that's Falsey?